ServerCore/Pages/Responses/Create.cshtml.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.EntityFrameworkCore;
+using ServerCore.DataModel;
+
+namespace ServerCore.Pages.Responses
+{
+    public class CreateModel : PageModel
```

Comment:
You're going to want to make this page (and all pages you add) Event-aware; it will standardize things and get the menu working properly.

https://github.com/PuzzleServer/mainpuzzleserver/wiki/Making-a-page-Event-aware